core/src/auctions.ts

```diff
@@ -83,7 +83,11 @@ const enrichAuctionWithMarketValues = async function (auction: Auction, network:
             marketUnitPrice,
             marketUnitPriceToUnitPriceRatio,
         };
-        const transactionGrossProfit = calculateTransactionGrossProfit(auctionWithMarketValues);
+        const transactionGrossProfit = calculateTransactionGrossProfit(
+            marketUnitPrice,
+            collateralToCoverDebt,
+            auction.approximateUnitPrice
+        );
```

Comment:
You already calculate `transactionGrossProfit` and `transactionGrossProfitDate` for each market inside `transactionGrossProfitDate`, why do you need to calculate it here again?

Reply:
Sorry, I meant `enrichAuctionWithMarketData`, not `transactionGrossProfitDate`. You have `marketData[marketId].transactionGrossProfit` which you can just take from there (if you would call `enrichAuctionWithMarketData` from within `enrichAuctionWithMarketValues`), instead of recalculating. As said above: recalculating might produce a different number and will be confusing